dotCMS/src/main/java/com/dotcms/rendering/velocity/services/PageRenderUtil.java

```diff
@@ -514,4 +510,40 @@ private String getPersonaTagToIncludeContent(final HttpServletRequest request, f
 
         return hasPersonalizations ? currentPersonaTag : MultiTree.DOT_PERSONALIZATION_DEFAULT;
     }
+
+    private static class ContainerUUIDPersona {
+        Map<String, List<String>> containerUuidPersona     = Maps.newHashMap();
```

Comment:
Codacy found an issue: [It is somewhat confusing to have a field name matching the declaring class name](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6340183)